cartridge/roles/crud-storage.lua

```diff
@@ -1,7 +1,9 @@
 local crud = require('crud')
+local stash = require('crud.common.stash')
 
 local function init()
     crud.init_storage()
+    stash.setup_cartridge_reload()
```

Comment:
I guess it is here (in the 'crud-storage' role), because we introduced the `crud.cfg()` function. Despite the fact that now there are no useful storage configuration options, we can add them in a future and it is meaningful to behave on storage in the same way as on router. Whether I understood the idea correctly?

Reply:
Exactly

Reply:
Okay so.